projects/docs-tools/src/modules/demo-page/demo-page.component.html

```diff
@@ -8,6 +8,22 @@
   >
     <ng-content select="sky-docs-demo-page-summary"></ng-content>
 
+    <sky-alert
+      *ngIf="
+        moduleTypeDefinitions?.hasPreviewFeatures ||
+        testingTypeDefinitions?.hasPreviewFeatures
+      "
+      class="sky-docs-demo-page-preview-alert sky-margin-stacked-lg"
+      >This component currently has preview inputs available that may not be
```

Comment:
Instead of "inputs" can we use "features" so that it accounts for inputs, outputs, services, etc.?

Reply:
Good call. Done